server/go/server.go

```diff
@@ -97,6 +98,33 @@ func handlePublicKey(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+func handleGetCheckoutSession(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		http.Error(w, http.StatusText(http.StatusMethodNotAllowed), http.StatusMethodNotAllowed)
+		return
+	}
+
+	keys, ok := r.URL.Query()["sessionId"]
```

Comment:
`key := r.FormValue("sessionId")`

it will be an empty string it it can’t find it.
https://pkg.go.dev/net/http/?tab=doc#Request.FormValue

Reply:
nice